idf_build_apps/manifest/manifest.py

```diff
@@ -69,6 +69,14 @@ def get_value(self, target: str, config_name: str) -> t.Any:
                 return content
         return self.default_clause
 
+    def __repr__(self) -> str:
```

Comment:
let's add some test on the string returned here, so we could make sure we don't break something later in the future.